chainregistry.go

```diff
@@ -211,6 +211,25 @@ func newChainControlFromConfig(cfg *config, chanDB *channeldb.DB,
 			cleanUp()
 			return nil, nil, err
 		}
+
+		// If the user provided an API for fee estimation, activate it now.
+		if cfg.NeutrinoMode.FeeURL != "" {
+			ltndLog.Infof("Using API fee estimator!")
+
+			estimator := lnwallet.NewWebAPIFeeEstimator(
+				cfg.NeutrinoMode.FeeURL,
+				cc.chainNotifier,
+				defaultBitcoinStaticFeePerKW,
+			)
+
+			if err := estimator.Start(); err != nil {
+				cleanUp()
+				return nil, nil, err
+			}
+			cc.feeEstimator = estimator
+			walletConfig.FeeEstimator = estimator
```

Comment:
is this necessary? Doesn't look like it is set for the other backends.

Reply:
fixed!

Reply:
Hm, I'm wondering... maybe it is a bug that it is not set for any backend?

We do set `walletConfig.FeeEstimator = cc.estimator` initially, but not sure if changing `cc.estimator` afterwards update the original reference? 🤔

Reply:
i checked and the original reference is not updated :( but also, I can't see `walletConfig.FeeEstimator` being used anywhere IIRC. I'll double check and prob add a commit removing it.

Reply:
That field isn't used anywhere so I added a commit removing it. Lmk if that's acceptable :thinking:

Reply:
Ah, even better! 💯